repos/base/mk/generic.mk

```diff
@@ -7,7 +7,7 @@
 # Collect object files and avoid duplicates (by using 'sort')
 #
 SRC_O  += $(addprefix binary_,$(addsuffix .o,$(notdir $(SRC_BIN))))
-SRC     = $(sort $(SRC_C) $(SRC_CC) $(SRC_ADA) $(SRC_RS) $(SRC_S) $(SRC_O))
+SRC     = $(sort $(SRC_C) $(SRC_CC) $(SRC_ADA) $(SRC_RS) $(SRC_S) $(SRC_O) $(SRC_ADS))
```

Comment:
Just nitpicking: It may make sense to add `SRC_ADS` right behind `SRC_ADA`.

Reply:
Maybe one could replace `SRC_ADA` with `SRC_ADB` also since this is distinguished anyway now?

Reply:
Sounds good to me, do ahead.

Reply:
`SRC_ADS` and `SRC_ADB` definitely make sense. We should change it and adapt the few Ada components now.